tests/e2e/e2e_controller_test.go

```diff
@@ -0,0 +1,39 @@
+package e2e
```

Comment:
Could you rename this test to `e2e_init_controller_test.go` or `e2e_controller_init_test.go`.

Also this test needs to be added to the list of tests in `e2e/README.md`